### ISSUE TITLE
【質問】データ管理画面の必要性

### ISSUE THREAD

**Issue by Keyhoh** [MayumiKamio/edubase#22]
## 内容

今後データが増えるにつれて、タグの追加や登録データの変更を行いたい状況が発生することが考えられます。
現状でもSpreadSheetに入力するとき、フォーマットに気を付けなければいけないと思います。
そこでデータの追加、変更、削除を行う管理画面があると便利だと思うのですが、必要性と優先順位、実装の手間はいかがでしょうか。

**Comment by ayazawa** [comment]
確かに、すでに情報の変更や追加を行っている状況なので、私は個人的にできる限り早く管理画面ができると良いともいます。管理画面がどのように作られるのかわからないのですが、すぐにできるものなのでしょうか。

**Comment by Keyhoh** [comment]
csvをインポートして編集・csv出力できる画面を作ってみます。
目標は木曜日で。

**Comment by MayumiKamio** [comment]
今後、先生方が直接データを入れていってくれるような機能を追加していきたいと思っていたので、中長期的にも使っていく機能になると思います。ありがとうございます！

**Comment by Keyhoh** [comment]
目標未達。。。週末に完成させます。

**Comment by MayumiKamio** [comment]
ありがとうございます…！

**Comment by MayumiKamio** [comment]
これって、前に話していたGoogleフォームと連動した方がいいですよね？

**Comment by Keyhoh** [comment]
はい、直近はGoogleフォームで入力して頂いて、入力したデータをcsvで連携して頂こうかと思っています。
Googleフォーム -> データをcsvに出力 -> csvをインポートしてデータを編集（ここを作っています） -> 編集後のデータをcsv出力 -> csvデータをDynamoDBに登録
というような連携を考えています。
今後の運用を考えると入力項目が増減したり、入力データの編集が発生すると思います。（多分現在でも発生しているかと）
そのときGoogleフォームやスプレッドシートではデータの編集が難しいと思いました。

**Comment by MayumiKamio** [comment]
承知しました！週末にGoogle formセットできるよう進めます。
＠ayazawa

こちら土曜のタスクに入れましょう！

**Comment by MayumiKamio** [comment]
自分たちが入力できる状態のフォームは完成。
https://forms.gle/cwzJTPuCn4CdRL95A

一般公開までのタスク
・外部の教員が入力するためのタグの説明を追加
・情報提供に同意してもらう際、どんな内容に同意してもらうかを決める必要あり
